shards/core/foundation.hpp

```diff
@@ -950,6 +950,7 @@ ALWAYS_INLINE inline void destroyVar(SHVar &var) {
   switch (var.valueType) {
   case SHType::None:
     return;
+  case SHType::Trait:
   case SHType::Type:
   case SHType::Table:
   case SHType::Set:
```

Comment:
You should remove the `default: break;` so it will warn about missing cases

Reply:
cannot really cos many blittable types pass from here, I will just add an assert